CONTRIBUTING.md

```diff
@@ -0,0 +1,92 @@
+# How to contribute
+
+Operator Controller is an Apache 2.0 licensed project and accepts contributions via GitHub pull requests (PRs).
+
+This document outlines some conventions on commit message formatting, contact points for developers, and other resources
+to help Operator Controller contributors.
```

Comment:
Do we require contributors to agree to the CNCF contributor license agreement?

Reply:
Right now we don't enforce it, could be a good thing to bring up at the WG meeting.